pydicom/dataelem.py

```diff
@@ -32,8 +31,7 @@
 
 from pydicom.valuerep import PersonNameUnicode
```

Comment:
I think we don't need `PersonNameUnicode` anymore - it is only used in Python 2.

Reply:
Removing that ... but you might need to look into values.py:
```
    # XXX - We have to replicate MultiString functionality
    # here because we can't decode easily here since that
    # is performed in PersonNameUnicode
```

I'm not sure what to do about that section -- if no longer needed, how much is removed?

Reply:
That is still needed, though I would remove the comment - it is a bit confusing, as other convertors do the same.